packages/target-web3-v1-test/test/DataTypesInput.test.ts

```diff
@@ -42,14 +49,23 @@ describe('DataTypesInput', () => {
 
     typedAssert(await contract.methods.input_stat_array(['1', '2', '3']).call(), ['1', '2', '3'])
     typedAssert(await contract.methods.input_stat_array([1, 2, 3]).call(), ['1', '2', '3'])
+    // TODO this fails due to an issue in web3 abi coder handling of inner BN:
```

Comment:
Let's link the web3 issue here and I think we can merge this, @krzkaczor what do you think?